indico/modules/events/surveys/models/surveys.py

```diff
@@ -197,3 +232,23 @@ def close(self):
         if self.state not in (SurveyState.active_and_clean, SurveyState.active_and_answered):
             raise IndicoError("Survey can't be closed")
         self.end_dt = now_utc()
+
+    def send_start_notification(self):
+        if not self.notifications_enabled or self.start_notification_sent:
+            return
+        addresses = self.start_notification_emails
+        if self.notify_participants:
+            addresses += self.event.getRegistrantsList() + self.event.getParticipation().getParticipantsList()
```

Comment:
I'm quite sure you did not test this code :volcano: 

I'd simply copy&paste the `EventReminder.all_recipients` property (and rename/adapt it). Once we redo participants+registrants we'll have to update it anyway (it'll be even simpler) so having almost the same code in two places now is not a problem.

```
    @property
    def all_recipients(self):
        """Returns all recipients of the notifications.

        This includes both explicit recipients and, if enabled,
        participants of the event.
        """
        recipients = set(self.recipients)
        if self.send_to_participants:
            event = self.event
            if event.getType() == 'conference':
                recipients.update(to_unicode(r.getEmail().strip().lower()) for r in event.getRegistrantsList())
            else:
                recipients.update(to_unicode(p.getEmail().strip().lower())
                                  for p in event.getParticipation().getParticipantList())
        recipients.discard('')  # just in case there was an empty email address somewhere
        return recipients
```